data/ayush24k.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "Ayush Anshu",
+  "DisplayStatsPublic": true,
```

Comment:
After the current update, you don't need to add this code. Check the [updated doc](https://linkfree.eddiehub.io/docs/quickstart)

```suggestion 
 
```